src/content/contributing/translation-program/resources/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Resources for translators
+lang: en
+description: Useful resources for ethereum.org translators
+sidebar: true
+---
+
+# Resources {#resources}
+
+You can find some useful guides and tools for ethereum.org translators, as well as translation communities and updates below.
+
+## Guides {#guides}
+
+- [Translation style guide](/contributing/translation-program/translators-guide/) _– instructions and tips for ethereum.org translators_
+- [Translation FAQs](/contributing/translation-program/faq/) _– frequently asked questions and answers about the ethereum.org Translation Program_
+- [Crowdin online editor guide](https://support.crowdin.com/online-editor/) _– an in-depth guide to using the Crowdin online editor and some of Crowdin's advanced features_
+- [Content buckets](/contributing/translation-program/content-buckets/) _– which pages are included in each content bucket of ethereum.org_
+
+## Tools {#tools}
+
+- [Microsoft Language Portal](https://www.microsoft.com/en-us/language)
+  _– useful for finding and checking the standard translations of technical terms_
+- [Linguee](https://www.linguee.com/)
+  _– search engine for translations and dictionary that enables searching by word or phrase_
+- [Proz term search](https://www.proz.com/search/)
+  _– database of translation dictionaries and glossaries for specialized terms_
+- [Eurotermbank](https://www.eurotermbank.com/)
+  _– collections of European terminology in 42 languages_
+
+## Communities {#communities}
+
+- [Language-specific Discord translation groups](https://discord.gg/6WX7E97)
```

Comment:
We should get in the habit of using our /discord/ link. 

```suggestion
- [Language-specific Discord translation groups](/discord/)
```

Reply:
@konopkja FYI we can track Discord link usage from our server by creating unique links for pages we want to track, but I assume Matomo provides us with this functionality already?

Reply:
matomo tracks external links usage so we can filter out and see clicks on a specific link